extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -92,6 +92,14 @@ public class OidcTenantConfig extends OidcCommonConfig {
     @ConfigItem
     public Optional<String> endSessionPath = Optional.empty();
 
+    /**
+     * The paths this tenant applies to. Static tenants can be resolved based on path-matching patterns. Path-matching is
```

Comment:
```suggestion
     * The paths which must be secured by this tenant.
```

Propose to simplify and explain the details in the docs - we can refer there to the HTTP Policy section etc